src/openrct2/object/RideObject.cpp

```diff
@@ -837,6 +837,7 @@ CarEntry RideObject::ReadJsonCar([[maybe_unused]] IReadObjectContext* context, j
             { "isWaterRide", CAR_ENTRY_FLAG_WATER_RIDE },
             { "isGoKart", CAR_ENTRY_FLAG_GO_KART },
             { "useDodgemCarPlacement", CAR_ENTRY_FLAG_DODGEM_CAR_PLACEMENT },
+            { "isWide", CAR_ENTRY_FLAG_WIDE },
```

Comment:
This is the kind of thing you should really discuss first. I’m not at all sure if I want this on the _vehicles_ of all things. I would agree it is something you want to have out of the track drawers, but the RTD seems like a better place.